apps/src/sites/studio/pages/signup.js

```diff
@@ -221,7 +194,7 @@ window.SignupManager = function (options) {
     ReactDOM.render(
       <div>
         <h5 style={{fontWeight: "bold"}}>
-          {showOptional ? i18n.schoolInformationOptionalHeader() : i18n.schoolInformationHeader()}
+          {i18n.schoolInformationHeader()}
```

Comment:
TODO: remove i18n.schoolInformationOptionalHeader from strings, should not be needed anymore